pkg/auth/rbac.go

```diff
@@ -201,6 +201,9 @@ func ReadRbacPolicy(dexEnabled bool) (policy map[string]*Permission, err error)
 
 // Checks user permissions on the RBAC policy.
 func CheckUserPermissions(rbacConfig RBACConfig, user *User, env, envGroup, application, action string) error {
+	if isEnvironmentIndependent(action) {
```

Comment:
Should envGroup then also be set to "*"?

Reply:
This action was handled on a different PR now. This change was removed.